zulipterminal/model.py

```diff
@@ -180,7 +180,7 @@ def get_message_ids_in_current_narrow(self) -> Set[int]:
             current_ids = self.index['all_starred']
         return current_ids.copy()
 
-    def _notify_server_of_presence(self) -> None:
+    def _notify_server_of_presence(self) -> Any:
```

Comment:
I think you should be able to tighten up this response type a little.

Reply:
Done.